src/main/java/org/jboss/hal/testsuite/page/RuntimePage.java

```diff
@@ -1,13 +1,14 @@
 package org.jboss.hal.testsuite.page;
 
 import org.jboss.hal.testsuite.fragment.runtime.ServerContextPicker;
+import org.jboss.hal.testsuite.page.config.ConfigurationPage;
 import org.jboss.hal.testsuite.util.Console;
 import org.openqa.selenium.WebElement;
 
 /**
  * Created by mvelas on 31.3.2014.
  */
-public abstract class RuntimePage extends BasePage {
+public abstract class RuntimePage extends ConfigurationPage {
```

Comment:
Please revoke this change. As we discussed face to face RuntimePage shouldn't extended Configuration change but rather methods from ConfigurationPage which can be shared should be pulled up to BasePage.